mis/add_one_row_to_tree.py

```diff
@@ -0,0 +1,128 @@
+#!/usr/bin/env python
+# coding: utf-8
+
+# https://leetcode.com/problems/add-one-row-to-tree/
+# 
+# 623. Add One Row to Tree
+# Medium
+# 
+# 330
+# 
+# 112
+# 
+# Favorite
+# 
+# Share
+# Given the root of a binary tree, then value v and depth d, you need to add a row of nodes with value v at the given depth d. The root node is at depth 1.
```

Comment:
Whatever the text editor you're using, please set the line limit to 80. It's universal everywhere for readability.

Reply:
Okay anh.